### PR TITLE
[IE Tests] Add new func tests to check IR version <> 10 readability

### PR DESCRIPTION
**Pull request by vurusovs**

### CODE REVIEW

File: inference-engine/tests/functional/ir_v7_reader/ir_v7_reader_test.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+#include <gtest/gtest.h>
+
+#include <inference_engine.hpp>
+
+
+TEST(IRReaderTest, ThrowIfIRVersionLessThan10) {
+    InferenceEngine::Core ie;
+
+    static char const *model = R"V0G0N(<net name="Network" version="7" some_attribute="Test Attribute">
```
Comment (ilya-lavrenov) [inline inference-engine/tests/functional/ir_v7_reader/ir_v7_reader_test.cpp:13]: how can we guarantee that IR v7 is not found?

File: inference-engine/tests/functional/ir_v7_reader/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME ieIRV7Tests)
+
+addIeTargetTest(
```
Comment (ilyachur) [inline inference-engine/tests/functional/ir_v7_reader/CMakeLists.txt:8]: IE Core loads readers in the runtime, we cannot avoid loading of v7 readers if we just create a new target.

**Comment by ilya-lavrenov** [comment]
@vurusovs wecan merge at least a test for compatibility with future versions.

**Comment by vurusovs** [comment]
@ilya-lavrenov  OK, I will update PR to merge soon

**Comment by ilya-lavrenov** [comment]
@vurusovs ping

File: inference-engine/tests/functional/ir_v7_reader/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME ieIRV7Tests)
```
Comment (iefode) [inline inference-engine/tests/functional/ir_v7_reader/CMakeLists.txt:6]: We need to update the CI because it's new target

Comment (ilya-lavrenov) [reply inference-engine/tests/functional/ir_v7_reader/CMakeLists.txt:6]: this target will not be commited

**Comment by ilya-lavrenov** [comment]
@vurusovs ping

**Comment by ilya-lavrenov** [comment]
Closed as inactive

**Pull request closed without merging**